src/lib.rs

```diff
@@ -273,17 +267,14 @@ where
                 height,
             };
 
-            let text = &flattened.text;
-            let item_style = text.style;
-
             let is_selected = state.selected == flattened.identifier;
             let after_highlight_symbol_x = if has_selection {
                 let symbol = if is_selected {
                     self.highlight_symbol
                 } else {
                     &blank_symbol
                 };
-                let (x, _) = buf.set_stringn(x, y, symbol, area.width as usize, item_style);
+                let (x, _) = buf.set_stringn(x, y, symbol, area.width as usize, Style::new());
```

Comment:
The `item.render` function can do styles, so I removed the additional style on the item itself. This means the indent isn't styled the same way then. I don't think this is really that bad.

9cb6dfd9cd91a03956936148ec19c5cc32fb59a9 removed the style as it also exists on Text and I prefer a single way of doing things rather than having styles everywhere.